packages/client/src/Manager.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
-class Jetpack_Client {
+namespace Automattic\Jetpack\Client;
+
+class Manager {
```

Comment:
Similar to what we did with Constants, should this live under the `Automattic\Jetpack` namespace and simply be called `Client`?

That way instead of:

```
use Automattic\Jetpack\Client\Manager as Jetpack_Client;
```

we'll have:

```
use Automattic\Jetpack\Client;
```

or:

```
use Automattic\Jetpack\Client  as Jetpack_Client;
```

(the last one if we insist on using the `Jetpack_` prefix.